### ISSUE TITLE
Transfer server fails checksum

### ISSUE THREAD

**Issue by Dr3Ame3r** [pterodactyl/panel#2661]
* Panel or Daemon: i don't really know, i think its daemon bug
* Version of Panel/Daemon: 1.0.2/1.0.1
* Server's OS:  Ubuntu 18.04 LTS
* Your Computer's OS & Browser: Firefox

Panel: `php -v` (the php version in use).

```
PHP 7.2.24-0ubuntu0.18.04.7 (cli) (built: Oct  7 2020 15:24:25) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.24-0ubuntu0.18.04.7, Copyright (c) 1999-2018, by Zend Technologies

```

Daemon: `uname -a` and `docker info` (your kernel version and information regarding docker)

`Linux xxxxxxo 5.8.16-xanmod1 #0~git20201018.dc69f04 SMP PREEMPT Sun Oct 18 12:32:07 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`
```

Client:
 Debug Mode: false

Server:
 Containers: 38
  Running: 6
  Paused: 0
  Stopped: 32
 Images: 13
 Server Version: 19.03.6
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version:
 runc version:
 init version:
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.8.16-xanmod1
 Operating System: Ubuntu 18.04.5 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 11.59GiB
 Name: xxxxxxxxo
 ID: 6JHE:7KTI:3BGI:HE6E:GQ24:QORX:O7VU:IQ6U:DVCB:67EE:XRAJ:QED3
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Username: dr3ame3r
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

**INFO:** Of what i've tested, cs1.6 fails with a symlink issue, samp moves flawlessly.
**INFO2:** Tests were performing monitoring the network traffic and interferences too, no errors or packet losses ocurred during this transfer. 

**To Reproduce**
1. Click on a server that's hosting a big server, let's say CS:GO
2. Click on Transfer Server and select options as you wish
3. tail -f /var/log/pterodactyl/wings.log -- No output is shown until the container archive is created in /var/lib/pterodactyl/archives/
4. After the archive has been created, Daemon tries to move the archive to the new server, it moves a part of the archive and then console shows following error:
```

ERROR: [Nov  5 09:41:06.123] checksum verification failed for archive server=de4c00a2-3bb1-4cc7-9dcc-d6304c8990bf
 INFO: [Nov  5 09:41:06.123] server transfer failed, notifying panel server=de4c00a2-3bb1-4cc7-9dcc-d6304c8990bf

```
5. Panel fails the transfer

Basically it moved only a part of the archive, in my case from 12.2G generated archive to only 11.3G

**Expected behavior**
1. Should not fail checksum and move the whole archive, we should see an output in wings daemon with the progress of the transfer or at least be able to track the movement process, output messages should be shown before the creation of the archive.
Status of the archive packaging should be shown too, percentages or whatever.

**Comment by matthewpi** [comment]
I'm going to close this as it seems nobody else has been able to reproduce this.

The checksum is of the final archive before and after it has been transferred to the other node, so something is going wrong when the target node requests the file from the source node.
